rplugin/python3/deoplete/sources/tabnine.py

```diff
@@ -84,8 +84,12 @@ def gather_candidates(self, context):
                 candidate['abbr'] = word
             else:
                 candidate['word'] = word
-            if result.get('detailed'):
-                candidate['menu'] = result['detailed']
+            if result.get('detail'):
+                candidate['menu'] = result['detail']
+            if result.get('documentation'):
+                info = result['documentation'][:256]
+                info = re.sub(r'\n\n', r'\n', info)
```

Comment:
Why remove double newlines?

Reply:
Because, it consumes preview lines...
OK.  I will remove it.

Reply:
Also why truncate at 256 characters? Both that and the removing of newlines seem like something that should be done by Deoplete so they can be configured by the user across all their sources.

Reply:
OK.  I will do that.